assets/js/directives/settings-form.directive.js

```diff
@@ -0,0 +1,37 @@
+angular
```

Comment:
@jtormey I wrapped these forms in a directive that handles the controller toggling to get rid of a lot of code reuse.